csrankings-c.csv

```diff
@@ -457,6 +457,7 @@ Chen Tian 0001,Nanjing University,https://cs.nju.edu.cn/tianchen,OYNQpFYAAAAJ
 Chen Yu 0001,Indiana University,https://psych.indiana.edu/directory/faculty/yu-chen.html,SA3IoXwAAAAJ
 Chen-Ching Liu,Washington State University,https://school.eecs.wsu.edu/people/faculty/chen-ching-liu,NOSCHOLARPAGE
 Chen-Nee Chuah,Univ. of California - Davis,https://www.ece.ucdavis.edu/~chuah/rubinet/people/chuah/bio.html,bZNRLNAAAAAJ
+Chenchen Liu,Univ. of Maryland - Baltimore County,http://cc-l.org/,sCLaaF0AAAAJ

```

Comment:
From DBLP:
> This is just a disambiguation page, and is not intended to be the bibliography of an actual person. Any publication listed on this page has not been assigned to an actual author yet. If you know the true author of one of the publications listed below, you are welcome to [contact us.](https://dblp.org/db/about/team.html)

This is also mentioned in the contributing notes, which you acknowledged:
>If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers ([dblp@dagstuhl.de](mailto:dblp@dagstuhl.de)) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

Please mail them per these instructions to request a disambiguation entry and open an issue; in the interim, please remove this faculty member's entry.